ApplicationLibCode/FileInterface/RifEclipseSummaryAddress.cpp

```diff
@@ -171,14 +171,40 @@ RifEclipseSummaryAddress::SummaryVarCategory RifEclipseSummaryAddress::identifyC
     auto category = RiuSummaryQuantityNameInfoProvider::instance()->categoryFromVectorName( strippedQuantityName );
     if ( category != SUMMARY_INVALID ) return category;
 
+    if ( strippedQuantityName[0] == 'N' ) return SUMMARY_NETWORK;
+    if ( strippedQuantityName[0] == 'W' ) return SUMMARY_WELL;
+
+    if ( strippedQuantityName[0] == 'R' )
+    {
+        // Taken from
+        // \ThirdParty\custom-opm-common\opm-common\src\opm\parser\eclipse\EclipseState\SummaryConfig\SummaryConfig.cpp
+
+        auto is_region_to_region = []( const std::string& keyword ) {
+            using sz_t = std::string::size_type;
+            if ( ( keyword.size() == sz_t{ 3 } ) && keyword[2] == 'F' ) return true;
+            if ( ( keyword == "RNLF" ) || ( keyword == "RORFR" ) ) return true;
```

Comment:
Appreciate that you take your time to formulate a detailed solution to the issues here. Thank you!

Reply:
To be able to reuse the text parsing and text compare functions in opm-common, it would have been useful to split the text parsing functions into a separate file.